impl/src/main/java/com/sun/faces/facelets/tag/ui/DecorateHandler.java

```diff
@@ -108,16 +107,12 @@ public void apply(FaceletContext ctxObj, UIComponent parent) throws IOException
             if (path.trim().length() == 0) {
```

Comment:
path.isBlank()

Reply:
not part of fix, but indeed candidate to be fixed project-wide in a new pr

Reply:
just because you were editing those files ... ;)